### PR TITLE
Chore/url generation

### PR DESCRIPTION
**Pull request by PorlyBe**
**Short Description:**

generate dev/prod urls

**Figma Link (optional):**
N/A

**Screenshots (optional):**
N/A

**This PR is blocked by or a continuation of:**
PR#

### CODE REVIEW

File: src/components/Boosts/Modals/Farm.tsx [hunk 1]
```diff
@@ -28,45 +29,27 @@ const Farm: FC = () => {
         </>
       )}
       <LinkButton
-        href={`https://verse.bitcoin.com/farms/eth/${
-          isWallet ? "?origin=wallet" : ""
-        }`}
-        {...(isWallet
```
Comment (micahpatng) [inline src/components/Boosts/Modals/Farm.tsx:34]: based on this code it should only open to new tab when `!isWallet`... right?

File: src/helpers/links.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+  query.append("coin", coin);
+  if (isWallet) query.append("origin", "wallet");
+
+  return `${swapURL}?${query.toString()}`;
+};
+
+export const generateBuyUrl = (isWallet: boolean, coin: "verse") => {
+  if (isWallet) {
+    const URL_DEEPLINK = `bitcoincom://buy/`;
```
Comment (micahpatng) [inline src/helpers/links.ts:62]: is there a reason why you're using back tick instead of `"`

**Review by micahpatng — CHANGES_REQUESTED** [review]
links are not working too...

File: src/components/Boosts/Modals/Farm.tsx (hunk 1 above)
Comment (PorlyBe) [reply src/components/Boosts/Modals/Farm.tsx:34]: ah you're right... fixed

File: src/helpers/links.ts (hunk 2 above)
Comment (PorlyBe) [reply src/helpers/links.ts:62]: consistency, lol

**Review by micahpatng — APPROVED** [review]
looks gooood!

**Pull request merged**